docs/DeveloperGuide.md

```diff
@@ -782,6 +782,90 @@ testers are expected to do more *exploratory* testing.
+      Expected: `eaddv` command runs successfully with the message `New volunteer added to event.`. However, the event roles are not updated in the UI since the volunteer's skills does not match the role.
+
+   3. Explore other cases and ensure that the skills of the volunteer matches the roles required for the event to add a volunteer to the event.
+
+### Tracking an event's materials and adding materials to an event
+
+1. Tracking event materials
+
+   1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`, followed by 
+      `eaddm eid/X m/50 packets` (where `X` is the index of the `Food packing` event)<br>
+      Expected: `eaddm` command runs successfully with the message `Amount of material updated in event: ...`. The `Food packing` event in the UI is updated with material `50 / 100 packets`.
+
+   2. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`, followed by
+      `eaddm eid/X m/100 packets` (where `X` is the index of the `Food packing` event)<br>
+      Expected: `eaddm` command runs successfully with the message `Amount of material updated in event: ...`. The `Food packing` event in the UI is updated with material `100 / 100 packets` that turns green.
+
+   3. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`, followed by
+      `eaddm eid/X m/100 bananas` (where `X` is the index of the `Food packing` event)<br>
+      Expected: `eaddm` command fails with the error message `Material bananas is not found in the event`.
+
+   4. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`, followed by
+      `eaddm eid/X m/100` or `eaddm eid/X m/banana` etc. (where `X` is the index of the `Food packing` event)<br>
+      Expected: `eaddm` command fails with the error message `Invalid command format!` with instructions on how to use `eaddm` properly.
+
+### Setting the maximum volunteer capacity
+
+1. Setting maximum volunteer capacity
+
+   1. Prerequisites: 
+      - At least 2 volunteers in the displayed volunteer list
+
+   2. Test case instructions in order (take note of the `vs/` parameter at the end):
+      1. `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/1`
+      2. `eaddv eid/X vid/1` (substitute `X` with the index of the `Food packing` event - e.g. `eaddv eid/1 vid/1`)
+      3. `eaddv eid/X vid/2` (substitute `X` with the index of the `Food packing` event - e.g. `eaddv eid/1 vid/2`)
```

Comment:
should change `substitute X with the` to `where X is the`

Reply:
good idea cos i did this for the subsequent ones and forgot change hahah

Reply:
fixed :D